.github/workflows/pull-requests.yml

```diff
@@ -0,0 +1,22 @@
+name: Pull Requests
+
+on: [pull_request]
```

Comment:
Clearly this only does PRs but maybe we want this to build on master as well?? We could rename the workflow and make a tweak i believe like this 

```
on:
  push:
    branches:
    - master
  pull_request:
    branches:
    - master
```

To run on prs against master and against any push to master.